lib/instabug_flutter.dart

```diff
@@ -248,7 +198,28 @@ class Instabug {
     await _channel.invokeMethod<Object>('show');
   }
 
+  /// Logs a user event with [name] that happens through the lifecycle of the application.
+  /// Logged user events are going to be sent with each report, as well as at the end of a session.
+  static void logUserEventWithName(String name) async {
```

Comment:
I know we renamed this class from `InstabugFlutter` to `Instabug` in a different PR, but since we're at it, let's also change the name of the file.